.gitignore

```diff
@@ -2,6 +2,8 @@
 __pycache__
 .pytest_cache
 
+/.venv-3.8
+
```

Comment:
Author's note: In most repositories, tools like pytest are already configured such that they don't try to operate in the files inside the virtualenv since we call them with explicit paths, e.g. `pylint edx_django_utils tests test_utils manage.py setup.py`.

A few repos would need to be adjusted to follow this pattern. These repos are already a problem for people who use in-repo virtualenvs.

There would also need to be some adjustment for code searching. Developers who are used to using `grep -nr ...` would need to instead write `grep -nr ... *`. Same for ack, although adding `--ignore-dir=match:/^.?venv-[0-9.]+$/` to one's `~/.ackrc` will prevent searching any such virtualenv. Those who use ripgrep would already be covered, as ripgrep ignores hidden dirs by default.